### DISCUSSION TITLE
用一句js小书签获取阿里云盘的refresh_token

### DISCUSSION THREAD

**Discussion by yunluo** [messense/aliyundrive-webdav#105] [Ideas]
下面js代码作为书签即可。
`javascript:(function(){prompt("%E6%82%A8%E7%9A%84%E9%98%BF%E9%87%8C%E4%BA%91%E7%9B%98refresh-token",JSON.parse(localStorage.getItem("token")).refresh_token)})();`

**Comment by hzl88688** [comment]
如何使用？

**Reply by hzl88688** [reply]
谢谢！用你的方法成功获取refresh_token

**Comment by shiba2046** [comment]
我用的这个
```javascript
javascript:prompt("阿里云token", JSON.parse(window.localStorage.token).refresh_token)
```

**Reply by Lovefish** [reply]
手机浏览器用不了吗？

**Comment by Lovefish** [comment]
手机浏览器用不了吗？还是我方法不对
